### PR TITLE
Select2 account selection controls have mixed initial state

### LINKED ISSUE #718: Regression: Select2 account selection controls have mixed initial state
1. Opportunities
 2. Create
 3. Click 'select existing'
 4. Click 'create'

Expected:
UI returns to initial state

Actual:
UI is in a mixed state - both controls are visible.

screen shot 2018-01-21 at 12 53 36 am

### PR DESCRIPTION
**Pull request by DmitryAvramec**
Fixes #718

### CODE REVIEW

File: app/helpers/accounts_helper.rb [hunk 1]
```diff
@@ -44,10 +44,17 @@ def account_select_or_create(form, &_block)
     content_tag(:div, class: 'label') do
       t(:account).html_safe +
         content_tag(:span, id: 'account_create_title') do
+<<<<<<< HEAD
+          "(#{t :create_new} #{t :or} <a href='#' onclick='crm.show_select_account(); return false;'>#{t :select_existing}</a>):".html_safe
+        end +
+        content_tag(:span, id: 'account_select_title') do
+          "(<a href='#' onclick='crm.show_create_account(); return false;'>#{t :create_new}</a> #{t :or} #{t :select_existing}):".html_safe
+=======
```
Comment (johnnyshields) [inline app/helpers/accounts_helper.rb]: Bad merge!

**Comment by johnnyshields** [comment]
@DmitryAvramec this PR needs some sort of test to confirm that reproduces the issue on current master and confirms the fix works.

**Comment by johnnyshields** [comment]
also, please squash your commits here

**Comment by CloCkWeRX** [comment]
Can just do squash-and-merge to clean it up on commit

**Pull request closed without merging**